src/ElvClient.js

```diff
@@ -637,9 +646,42 @@ class ElvClient {
     ethProvider.pollingInterval = 250;
     this.signer = ethProvider.getSigner();
     this.signer.address = await this.signer.getAddress();
+    await this.SetSignDigestFromWeb3Provider({provider});
```

Comment:
this should be conditional, shouldn't it?

Reply:
conditional on what?

Reply:
There's a method called SetSignerFromWeb3Provider that's used for metamask. Is this unrelated to that?

Reply:
ah, yea, this is called from inside SetSignerFromWeb3Provider().  i could have just inlined it there, but, thought maybe someone might want to use it separately.  can rename to SetSignDigest() in the next PR.